cmd/bridge/main.go

```diff
@@ -84,6 +80,7 @@ func main() {
 	fK8sAuthBearerToken := fs.String("k8s-auth-bearer-token", "", "Authorization token to send with proxied Kubernetes API requests.")
 
 	fLogLevel := fs.String("log-level", "", "level of logging information by package (pkg=level).")
+	fVerbosity := fs.String("v", "", "level of verbosity of logging from level 0-5.")
```

Comment:
This should come already from `klog` right?

https://github.com/kubernetes/klog/blob/master/klog.go#L429

If `klog.InitFlags()` is supposed to give us this, I'm suspicious the wiring is incorrect & its going to bite us later.

Reply:
Yes, it does get handled in klog InitFlags but 1) console throws an error if we do not have the flag defined and 2) because we don't have pflags (like in console-operator: https://github.com/openshift/console-operator/blob/master/cmd/console/main.go#L30), the values do not get transferred for some reason.

Reply:
isn't it a bit odd that we are setting both, `log-level` and `verbosity`? Thought that log-level describes the logging verbosity

Reply:
isn't it a bit odd that we are setting both, `log-level` and `verbosity`? Thought that log-level describes the logging verbosity

Reply:
Yes, these custom flags need to be removed, `klog` already provides what is needed for its own configuration.

Reply:
You shouldn't need `pflags` either, see the [comment i left](https://github.com/openshift/console/pull/1862#issuecomment-563334473) regarding a minimal `klog` setup (this code runs & you can restart changing the passed in flag to get different outputs).

Reply:
`log-level` is being kept for now for backwards compatibility and we will remove it in the future.  As I mentioned before about the flags, console throws a fatal error if you try to pass an unknown flag.  The flag value from klog does not get sent over to the console side of things... or at least not automatically.  We can pair up on this to try some other things if you'd like @benjaminapetersen @jhadvig

Reply:
I can pass the flagset of console into the klog init function.  Maybe that will work.  I will try it out.

Reply:
Ok, passing the console's flagset into the init function works.  I will update the code.